### DISCUSSION TITLE
HMIP-DLD zeigt abgeschlossen an obwohl ich nur die Türe öffne

### DISCUSSION THREAD

**Discussion by freigeistmaschine** [SukramJ/aiohomematic#1797] [Q&A]
Ich habe einen Bug gefunden. Wenn ich das DLD über Home Assistent öffne oder über Homekit, was ja auch über Home Assistent läuft also quasi über die HMIP Intergration, dann wird in der Log nicht nur aufgeschlossen, sondern auch eine Millisekunde vorher "abgesperrt" angezeigt, siehe Screenshot.

Habe gerade mein Kumpel gefragt, der auch das gleiche Setup hat, bei ihm ist es auch so?

Wenn ich die Türe also das DLD über die Raspberrymatic Weboberfläche öffne, steht das abgeschlossen nicht dabei, sondern nur aufgesperrt.

Aufgefallen ist mir der Fehler, weil mir die Apple Home App 2 Notifications geschickt hat, einmal abgeschlossen und direkt danach aufgeschlossen.

<img width="998" alt="Bildschirmfoto 2024-10-26 um 16 17 45" src="https://github.com/user-attachments/assets/be53c5b2-53b5-43bb-ab00-72d97e01abee">

![WhatsApp Image 2024-10-26 at 14 44 23 (1)](https://github.com/user-attachments/assets/36e02f50-9ccb-4027-bd73-6a3e96e03f36)

**Comment by SukramJ** [comment]
Bitte nicht spammen #1796
